Automattic/vip-go-mu-plugins
# Search: Change add_version() to account for failed put_mapping

## Description
Currently, `add_version()` assumes that `put_mapping()` will succeed. We should return a WP_Error if it doesn't though.

## Changelog Description

### Plugin Updated: Enterprise Search

Search: Account for failed put_mapping call in add_version()
## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Go to `search/elasticpress-next/includes/classes/Elasticsearch.php` and change `put_mapping()` to return `false`
2) Attempt to create new index version `wp vip-search index-versions add post` and see failed error message:
```
Error: Unable to properly create new index with mapping
```
